projects/chaos/src/sequencer_fsm.c

```diff
@@ -154,6 +154,8 @@ static const SequencerEventPair s_charge_events[] = {
     .response = SEQUENCER_NO_RESPONSE },
   { .raise = { .id = CHAOS_EVENT_GPIO_CHARGE, .data = SEQUENCER_EMPTY_DATA },
     .response = SEQUENCER_NO_RESPONSE },
+  { .raise = { .id = CHAOS_EVENT_DELAY_MS, .data = 1000 },
```

Comment:
You should see if this delay is still necessary now that we aren't faulting immediately.